tests/test_map_control.py

```diff
@@ -0,0 +1,125 @@
+
+# from unittest.mock import patch
+import pytest
+
+
+from awpy.analytics.map_control import frame_map_control_metric
+
+
+class TestMapControl:
+    """Class to test the map control-related functions."""
+
+    def setup_class(self):
+        """Setup class by defining custom Map Control object"""
+
+        self.fake_alive_player = {
+            "x": -42.51047897338867,
+            "y": 868.4791870117188,
+            "z": 54.92256546020508,
+            "isAlive": True,
+        }
+        self.fake_frames = {
+            "map_control_sanity_t_control": {
+                "t": {
+                    "players": [
+                        self.fake_alive_player.copy(),
+                        self.fake_alive_player.copy(),
+                        self.fake_alive_player.copy(),
+                        self.fake_alive_player.copy(),
+                        self.fake_alive_player.copy(),
```

Comment:
Could use list comprehension here.

`[self.fake_alive_player.copy() for _ in range(5)]`

Reply:
Addressed for next commit on branch